tests/src/test/java/alluxio/master/journal/ufs/package-info.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+/**
+ * Tests for the UFS journal. These tests require an under file system but not a local Alluxio
+ * cluster. Ideally they will become unit tests if we can mock the under file system or allow local
```

Comment:
would it cause problems to add LocalUnderFileSystem as a test dependency for common? It would be nice to have everything under `tests/` be an integration test

Reply:
Yeah I was planning on doing that or rewriting the tests with a proper UFS mock. Wanted to limit the changes here to just relocation.

Reply:
sg